packages/ui/src/CampaignRewards/CampaignRewardsComp.tsx

```diff
@@ -25,18 +27,42 @@ const RewardsCompSmall: React.FC<CampaignRewardsCompProps> = ({ rewardsPool, hig
   )
 }
 
-const TooltipMessage = ({
-  platform,
-  description,
-  dashboardLink,
-}: {
-  platform: string
-  description: string
-  dashboardLink: string
-}) => {
+const TooltipMessage = ({ rewardsPool }: { rewardsPool: RewardsPool }) => {
+  const { campaignName, platform, description, dashboardLink, campaignStart, campaignEnd } = rewardsPool
+
+  const options: Intl.DateTimeFormatOptions = {
```

Comment:
It would be nice to take this definition out of the component so that it is only defined once, rather than being created on every render. It won't affect performance now, but if in the future this variable will be in dependencies, it will lead to unnecessary re-rendering.